src/System.Drawing.Common/src/System/Drawing/FontFamily.cs

```diff
@@ -185,13 +185,19 @@ private void Dispose(bool disposing)
         /// <summary>
         /// Returns the name of this <see cref='FontFamily'/> in the specified language.
         /// </summary>
-        public string GetName(int language)
+        public unsafe string GetName(int language)
         {
             // LF_FACESIZE is 32
-            var name = new StringBuilder(32);
+            Span<char> name = stackalloc char[32];
+            fixed (char* ptr = name)
```

Comment:
stackalloc'ing into a span and then fixing it doesn't really buy anything here.  It seems like this whole thing can just be done using the code that was already there, but without the StringBuilder, which appears to have been entirely superfluous:
```C#
char* ptr = stackalloc char[32]; // LF_FACESIZE
int status = SafeNativeMethods.Gdip.GdipGetFamilyName(new HandleRef(this, NativeFamily), ptr, language);
return Marshal.PtrToStringUni((IntPtr)namePtr);
```

Reply:
looks like what we have in mono for this method 🙂 https://github.com/mono/mono/blob/90d11817f8ed9ebb3dfa982f83a4866c57df6ba6/mcs/class/System.Drawing/System.Drawing/FontFamily.cs#L57-L60